webpack.config.js

```diff
@@ -0,0 +1,28 @@
+const path = require('path');
+var HtmlWebpackPlugin = require('html-webpack-plugin');
+
+var HTMLWebpackPluginConfig = new HtmlWebpackPlugin({
+  template: __dirname + '/public/index.html',
+  filename: 'index.html',
+  inject: 'body'
+});

```

Comment:
Typically, after you configure this plugin you would include it as part of a plugins key on the exports.

``` javascript
plugins: [HTMLWebpackPluginConfig],
```